[IMP] accounting: belgium localization updates (#18312)
odoo/documentation: content/applications/finance/fiscal_localizations/belgium.rst

Belgium
=======

.. _localizations_belgium/configuration/modules:

Modules
=======

The following modules are installed automatically with the Belgian localization:

.. list-table::
   :header-rows: 1
   :widths: 25 25 50

   * - Name
     - Technical name
     - Description
   * - :guilabel:`Belgium - Accounting`
     - `l10n_be`
     - Belgian :ref:`fiscal localization package <fiscal_localizations/packages>`, complete with
       the Belgian chart of accounts, taxes, tax report, and fiscal positions
   * - :guilabel:`Belgium - Accounting Reports`
     - `l10n_be_reports`
     - Module providing Belgian accounting reports
   * - :guilabel:`Belgium - Accounting Reports - SMS`
     - `l10n_be_report_sms`
     - Bridge module between Belgian accounting and SMS
   * - :guilabel:`Belgium - Accounting Reports Client Nihil`
     - `l10n_be_reports_client_nihil`
     - Extension for accouting returns in Belgium
   * - :guilabel:`Belgian Intervat & Myminfin Edi`
     - `l10n_be_intervat`
     - Integration with Intervat and MyMinFin APIs, allowing to send and receive electronic VAT

>        declarations

Reviewer: ```suggestion
     - Integration with Intervat and MyMinfin APIs, allowing the sending and receipt of electronic VAT
       declarations
```

Reviewer: FYI: In English, 'allow + infinitive' isn't gramatically correct. The above is one way ot rephrasing it without resorting to the passive (i.e., allowing electronic VAT declarations to be sent and received)

You can reach the :guilabel:`Chart of accounts` by going to :menuselection:`Accounting -->
Configuration --> Accounting: Chart of Accounts`.

The Belgian chart of accounts includes pre-configured accounts as described in the :abbr:`PCMN (Plan

> Comptable Minimum Normalisé)`. To add a new account, click :guilabel:`New`. A new line appears. Fill

Reviewer: A bit of a general question, for my own understanding: when we refer to something like the PCMN, is it deliberate that we only mention the French version, and not the Dutch one? (It's just something I remember always checking up when I used to translate blog posts for Accountable into English, and it felt more complete/fair/user-friendly to provide both the French and Dutch names of things ... even if a bit messy and long sometimes. (And we never put the German name 🙉 ) Similar question then for other countries with more than one official language.

Reviewer 2: I'm not aware of any guideline or policy, it is in French because I picked it up where Jon left off and he wrote it only in French. It is a good point though, I'll include the Dutch name and we can also discuss this at the next team meeting

Reviewer: Sounds good. I'll create a task in the team meeting project.

.. seealso::
   :doc:`Taxes <../accounting/taxes>`

.. _belgium/intervat:

Intervat
--------

.. note::
   - Make sure the :guilabel:`Belgian Intervat & Myminfin Edi` (`l10n_be_intervat`) module is
     :doc:`installed <../../general/apps_modules>` on your database.
   - When submitting your VAT returns, make sure to use XML or VAT format files. These are the only
     file formats accepted by Intervat.

`Intervat <https://financien.belgium.be/en/E-services/intervat>`_ is the electronic platform used to
submit VAT returns to the Belgian tax authorities. You can file your returns directly from Odoo
using the Intervat integration.

To set this up, open the **Accounting** app, go to :menuselection:`Configuration --> Settings`, and
scroll down to the :guilabel:`Taxes` section. From there, select a :guilabel:`Server Mode`, enter
your :guilabel:`VAT Number`, and select an :guilabel:`Accounting` firm if applicable.

.. seealso::
   :doc:`../accounting/reporting/tax_returns`

.. _belgium/non-deductible:

Non-deductible taxes
--------------------

In Belgium, some taxes are :ref:`not fully deductible <belgium/disallowed-expenses>`, such as taxes

> on the maintenance of cars. This means a part of these taxes is considered as an expense.

Reviewer: Maybe this would be better, since I guess the whole amount is still a business expense, but some of it is disallowed:

```suggestion
In Belgium, some taxes are not fully deductible, such as taxes on the maintenance of cars. This means a part of these taxes is considered as a :ref:`non-deductible expense <belgium/disallowed-expenses>`.
```

   .. image:: belgium/deductible-tax.png
      :alt: Example of not-fully deductible tax

> Vehicle tax deduction

Reviewer: ```suggestion
Vehicle tax deductibility
```

I'm wondering if this is a better heading ...

Reviewer 2: It is :)

---------------------

.. note::
   To see the tax deductibility of a vehicle, the **Belgium - Disallowed Expenses Fleet**
   (`l10n_be_account_fiscal_categories_fleet`) module must be :doc:`installed
   <../../general/apps_modules>`.

A vehicle's tax deductibility rate varies depending on its type (car or bicycle) and several
factors, such as fuel type, CO2 emissions, engine power, etc.

> To view the **tax deductibility percentage** for a specific vehicle, open the **Fleet** app,

Reviewer: ```suggestion
To view the **tax deductibility percentage** for a specific model, open the **Fleet** app,
```

Reading this again, I'm thinking this should be 'model' rather than 'vehicle' right?

navigate to :menuselection:`Configuration --> Models`, and select a vehicle model. Locate the
:guilabel:`Tax Deduction` field, which is found under the :guilabel:`Engine` section for cars, or
the :guilabel:`Vehicle Information` section for bicycles.

.. important::
   The :guilabel:`Tax Deduction` field is strictly **informative** and is computed automatically
   based on the vehicle's specifications. It is **not** used for any automated calculations within
   the **Accounting** app and should not be confused with the :ref:`tax rate deductibility
   <belgium/non-deductible>` used on :ref:`tax grids <accounting/tax-returns/tax-grids>`. Instead,
   your accountant can reference this field to manually apply the correct deductible rate to

>    invoices or disallowed expenses.

Reviewer: Add link to disallowed expenses section? (And refer to this section from that one?)

.. seealso::
   :ref:`Vehicle models <fleet/models>`

Intervat

> --------

Reviewer: Could it make sense for this to be the first sub-topic under taxes? Feels a bit strange to have it come after the others.

.. note::

>    - Make sure the :guilabel:`Belgian Intervat & Myminfi Edi` module is :doc:`installed

Reviewer: ```suggestion
   - Make sure the :guilabel:`Belgian Intervat & Myminfin Edi` (`l10n_be_intervat`) module is :doc:`installed
```

     <../../general/apps_modules>` on your database.

>    - When submitting your tax returns, make sure to use XML or VAT format files. These are the only

Reviewer: ```suggestion
   - When submitting your VAT returns, make sure to use XML or VAT format files. These are the only
```

I guess if we're referring to Intervat, then we're only talking about VAT returns, right?

     file formats accepted by Intervat.

> Intervat is the electronic platform used to submit tax returns to the Belgian tax authorities. You

Reviewer: ```suggestion
Intervat is the electronic platform used to submit VAT returns to the Belgian tax authorities. You
```

can file your returns directly from Odoo using the Intervat integration.

To set this up, open the **Accounting** app, go to :menuselection:`Configuration --> Settings`, and

> scroll down to the :guilabel:`Taxes` section. From there, select a :guilabel:`Server Mode`, enter

Reviewer: Just:

```suggestion
To set this up, go to :menuselection:`Accounting --> Configuration --> Settings` and scroll down to the :guilabel:`Taxes` section. From there, select a :guilabel:`Server Mode`, enter
```

Reviewer 2: The reason I prefer to write **Accounting** app separately is because there's already an "Accounting" menu within the Accounting app, and if the users look for a specific information while already having the Accounting app open, they might be confused and look in the wrong menu. We can discuss this to see how to bring this in line with the rest of the apps, potentially :)

.. _belgium/disallowed-expenses:

Disallowed expenses report
--------------------------
Disallowed expenses
-------------------

.. note::
   Fiscal categories are only accessible when :doc:`developer mode <../../general/developer_mode>`
   is enabled.

**Disallowed expenses** are expenses that can be deducted from your accounting result but not from

> your fiscal result.

Reviewer: I indeed found this sentence not the most user-friendly, but maybe it's still good to have a line about disallowed expenses? (My suggestion was meant to be simple, but it got a bit long ... let's discuss?)

Disallowed expenses are expenses that are not fully tax-deductible. For example, in Belgium, restaurant costs are 69% tax-deductible, with 31% considered a disallowed expense on which income tax or corporate tax must be paid.

Reviewer 2: I added this on line 216:

Disallowed expenses are business costs that are not fully tax-deductible. While they count as business expenses in your accounting, tax laws do not allow you to use them to lower your taxable income.

> **Accounting** app and going to :menuselection:`Reporting --> Fiscal Report`. This report provides

Reviewer: ```suggestion
The **Fiscal Report** tracks disallowed expenses and is available by going to :menuselection:`Accounting --> Reporting --> Fiscal Report`. This report provides
```

Just to be a little more concise and align with some other wording on this page.

Reviewer 2: I made it slightly less of a mouthful

The report is generated based on **fiscal categories**, which can be managed by going to

> :menuselection:`Configuration --> Fiscal Categories`. While several default categories are provided,

Reviewer: ```suggestion
:menuselection:`Accounting --> Configuration --> Fiscal Categories`. While several default categories are provided,
```

I think it's better to add 'Accounting' again just to be complete, no?

Reviewer 2: I added the "Accounting app" step to be complete. The reason I didn't include it in the menuselection is because there's already an "Accounting" menu within Accounting, and if users are reading the doc and are already in the accounting app, it could be confusing for them

> you can create new ones by clicking :guilabel:`New`. Categories themselves do not hold rates.

Reviewer: ```suggestion
you can create new ones by clicking :guilabel:`New`. Categories themselves do not contain rates.
```

Could 'contain' be better than 'hold'? Unless there's a specific reason for 'hold' in an accounting context?

Instead, you must link a category to an account by opening the fiscal category and specifying the
target accounts in the :guilabel:`Related Account(s)` field. Optionally, you can restrict the

> category to a specific company by selecting it from the :guilabel:`Company` drop-down menu.

Reviewer: ```suggestion
Instead, categories are linked to accounts via the :guilabel:`Related Account(s)` field. In a multi-company environment, you can restrict the category to a specific company by selecting it from the :guilabel:`Company` dropdown menu.
```

.. tip::
   If the **Fleet** app is installed, check the :guilabel:`Requires a Vehicle` box where applicable.
   This makes selecting a vehicle mandatory when logging a vendor bill.

> To set a rate on an account, go to :menuselection:`Configuration --> Chart of Accounts` and open the

Reviewer: ```suggestion
To set a rate on an account, go to :menuselection:`Accounting --> Configuration --> Chart of Accounts` and open the
```

account linked in the previous step. Under the :guilabel:`Fiscal Rates` tab, click :guilabel:`Add a
line` and enter a :guilabel:`Start Date` and :guilabel:`Fiscal Rate (%)`.

> Moving forward, whenever an expense is recorded using this account, the disallowed expense amount is

Reviewer: ```suggestion
Thereafter, whenever an expense is recorded using this account, the disallowed expense amount is
```

automatically calculated based on the rate specified in the :guilabel:`Fiscal Rates` tab.

.. tip::
    You can add multiple rates for different dates. The system will automatically apply the correct
    rate based on the date the expense is recorded.

> Let's take an example reflecting **restaurant** and **car expenses**.

Reviewer: The tone and use of contraction here feel a bit out of place, no?

Maybe you could just:

- drop this line
- (If you really want to have these as TOC tree items rather than formatted as examples) change the titles to, respectively:
     - Example: Restaurant expenses
     - Example: Vehicle expenses

Reviewer 2: You're right. I only modified and expanded on the previous doc so I skimped what was unchanged, but the tone doesn't match the documentation and the sentence doesn't serve any purpose itself :)

.. _belgium/disallowed-expenses:

Disallowed expenses
-------------------

.. note::
   Fiscal categories are only accessible when :doc:`developer mode <../../general/developer_mode>`

>    is enabled.

Reviewer: I'm wondering now if we need to have this note here ... it's a little distracting for what's essentially a configuration step we can just include in the text, no? See below for suggestion ...

Reviewer 2: Removed it and integrated it to the text below

Disallowed expenses are business costs that are not fully tax-deductible. While they count as
business expenses in your accounting, tax laws do not allow you to use them to lower your taxable
income.

The **Fiscal Report** tracks disallowed expenses and is available by opening the **Accounting** app
and going to :menuselection:`Reporting --> Fiscal Report`. This report provides real-time financial
results and allows for periodic adjustments.

The report is generated based on **fiscal categories**, which can be managed by opening the

> **Accounting** app and going to :menuselection:`Configuration --> Fiscal Categories`. While several

Reviewer: ```suggestion
The report is generated based on **fiscal categories**, which can be managed by opening the
**Accounting** app, activating :ref:`developer mode <developer-mode>` and going to :menuselection:`Configuration --> Fiscal Categories`. While several
```

default categories are provided, you can create new ones by clicking :guilabel:`New`. Categories
themselves do not contain rates. Instead, categories are linked to accounts via the
:guilabel:`Related Account(s)` field. In a multi-company environment, you can restrict the category

> to a specific company by selecting it from the :guilabel:`Company` dropdown menu.

Reviewer: How about adding the 'In a multi-company environment, ...' to the tip just below? Like for having the Fleet app, multi-company is also a less common scenario I guess.

.. tip::
   If the **Fleet** app is installed, check the :guilabel:`Requires a Vehicle` box where applicable.
   This makes selecting a vehicle mandatory when logging a vendor bill.

.. _belgium/set-fiscal-rate:

To set a rate on an account, open the **Accounting** app and go to :menuselection:`Configuration -->
Chart of Accounts` and open the account linked in the previous step. Under the :guilabel:`Fiscal
Rates` tab, click :guilabel:`Add a line` and enter a :guilabel:`Start Date` and :guilabel:`Fiscal
Rate (%)`.

Thereafter, whenever an expense is recorded using this account, the disallowed expense amount is
automatically calculated based on the rate specified in the :guilabel:`Fiscal Rates` tab.

.. tip::
    You can add multiple rates for different dates. The system will automatically apply the correct
    rate based on the date the expense is recorded.

.. _belgium/restaurant-expenses:

Restaurant expenses
~~~~~~~~~~~~~~~~~~~

**Restaurant** expenses are 69% deductible. To configure this, create a new :guilabel:`Fiscal
Category`, specify its :guilabel:`Related Account(s)`, then set the relevant :ref:`fiscal rates

> <belgium/set-fiscal-rate>` on the respective related account(s).

Reviewer: In hindsight, maybe it's silly that I suggested linking back to where you explain setting fiscal rates ... I didn't suggest you link back up to where you explain creating the fiscal categories in the first place. 

So I don't know ... what do you think? You literally _just_ explained both right above, so maybe no need for links at all.

Reviewer 2: From my point of view, users don't read the documentation beforehand. If they read the documentation, it's because they're facing an issue within Odoo. From this stand-point, they are most likely to search only what they need or want, they're not going to read the whole page if it's unrelated to the issue they are facing. Therefore, if they have a question regarding restaurant expenses in Belgium, this section might be the first thing they read. So it's fine like this for me.

Reviewer: Works for me!

.. _belgium/vehicle-split:

Vehicle expenses: vehicle split
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The :ref:`deductibility percentage <belgium/vehicle-tax-deductibility>` expenses of vehicles varies

> by vehicle and must be defined for each vehicle in Odoo. To do this, open the :menuselection:`Fleet`

Reviewer: I think something went wrong with the first sentence 😅 

How about:

```suggestion
The :ref:`deductibility percentage <belgium/vehicle-tax-deductibility>` of vehicles varies by model and must be defined for each vehicle in Odoo. To do this, open the :menuselection:`Fleet`
```

Reviewer 2: I didn't use the word "Model" because there's a 'Model' menu option and a 'Model' tab in the Fleet app, and neither allow you to configure a tax deductibility. Perhaps I'm being over-careful here though, as there's no :guilabel: used for "Model". I rewrote it differently. What do you think?

Reviewer: Yep, your rewrite is fine by me!

Restaurant expenses
~~~~~~~~~~~~~~~~~~~

In Belgium, 69% of **restaurant** expenses are deductible. To configure this, create a new **fiscal

> category** and specify its :guilabel:`Related Account(s)` and :guilabel:`Fiscal rates`.

Reviewer: Maybe change the wording to match how you talked about vehicle tax deduction earlier?

```suggestion
In Belgium, the tax deductibility percentage for **restaurant** expenses is 69%, meaning 31% is considered a disallowed expense. To configure this, create a new :guilabel:`Fiscal Category`, specify its :guilabel:`Related Account(s)`, then set the relevant :guilabel:`Fiscal Rates` on the respective related account(s).
```

Also, right now it's a little unclear that the Fiscal rates are on the related account and not on the fiscal category itself. My wording may not be ideal, but it's a suggestion. And could it be an idea to add an anchor on the below paragraph and link to it from '... set the relevant :guilabel:`Fiscal Rates` ...'.

Lastly, here and in the next example, do we need the 'In Belgium', since we're on a page only about Belgium?

.. _belgium/vehicle-split:

> Car expenses: vehicle split

Reviewer: ```suggestion
Vehicle expenses: vehicle split
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~

> In Belgium, the deductible percentage varies per car and must be defined for each individual

Reviewer: By 'deductible percentage' (or my suggested rewording below), do we mean what we just explained above in 'Vehicle tax deduction'?

Reviewer 2: Yes. From my understanding, both the car itself *and* expenses (e.g., maintenance) for that car have some level of deductibility rates, and so are therefore two separate topics, but both fall into disallowed expenses. I never had a company, so perhaps I'm wrong on this one. Please feel free to point it out if it is the case, I trust you know this topic better than I do :)

vehicle. To do this, open the :menuselection:`Fleet` app and select a vehicle. In the :guilabel:`Tax

> info` tab, locate the :guilabel:`Fiscality` section and click :guilabel:`Add a line` under the

Reviewer: ```suggestion
In Belgium, the tax deductibility percentage varies per vehicle model and must be defined for each individual
vehicle. To do this, open the :menuselection:`Fleet` app and select a vehicle. In the :guilabel:`Tax Info` tab, locate the :guilabel:`Fiscality` section and click :guilabel:`Add a line` under the
```

Should it be 'model' vs 'vehicle' in that first sentence? (Though now I'm confused as the Tax Deduction for this _model_ is:

<img width="470" height="784" alt="Image" src="https://github.com/user-attachments/assets/73f2c78a-2aed-445b-a306-4d47a6dcd675" />

While the Tax Deduction for this specific vehicle of that model in the same runbot is:

<img width="543" height="561" alt="Image" src="https://github.com/user-attachments/assets/bb14721a-a95e-4f10-b8f1-0760a4bde460" />

Also, about this part: '... and must be defined for each individual
vehicle'. Do we mean that '... and must be defined for each individual vehicle _in Odoo_'?

Reviewer 2: As discussed, it seems models and vehicles (so long as it's the same vehicle) have the same tax deduction percentage. However, I did specify **in Odoo** in the introductory line :)

:guilabel:`Fiscal Deductibility` field. Add a :guilabel:`Start Date` and a :guilabel:`Non-deductible

> (%)`. The expenses for all cars will still be recorded to the same account.

Reviewer: I'm not sure I get this _(Edited to add: specifically line 252)._ But should it maybe be a note? Are you adding it because something changed with respect to this? Or because you think there might be a doubt?

Reviewer 2: I want to make it clear and explicit where the expenses are recorded, but perhaps it's obvious in itself for company owners (I'm not personally familiar with disallowed expenses at all).

I changed the sentence to make it even clearer, but as discussed, let me know if it's actually not needed/necessary in the documentation

> (%)`. The expenses for all cars will still be recorded to the same account.

Reviewer: I'm confused. This 'Non-deductible (%)' ... is this based on the 'Tax Deduction' field we explained in the 'Vehicle tax deduction' section?

Reviewer 2: Indeed, you input the non-deductible % rate (so the difference of what is indicated in the 'Tax Deduction' field to make 100%) for the different dates.

I rewrote it a bit to make it clearer

When creating a vendor bill for car expenses, you can link each expense line to a specific car by

> filling in the :guilabel:`Vehicle` column (accessible under :guilabel:`Amount`

Reviewer: ```suggestion
When creating a vendor bill for vehicle expenses, you can link each expense line to a specific vehicle by
filling in the :guilabel:`Vehicle` column (accessible under :guilabel:`Amount`
```

I think we're better using 'vehicle' rather than car, no?

> :icon:`oi-settings-adjust`). This ensures the correct non-deductible percentage is applied.

Reviewer: Suggestion to edit this, since the Account label and slider icon are not 'together'.

```suggestion
filling in the :guilabel:`Vehicle` column (accessible by clicking the 
:icon:`oi-settings-adjust` :guilabel:`(slider)` icon beside :guilabel:`Amount`, then selecting :guilabel:`Vehicle`. This ensures the correct non-deductible percentage is applied.
```

.. image:: belgium/car-bill.png
   :alt: Disallowed expenses of car bill

To view the specific rates and disallowed amounts calculated for each car, enable the
:guilabel:`vehicle split` filter (located under :icon:`fa-sliders` :guilabel:`Posted Entries`)

> within the fiscal report.

Reviewer: ```suggestion
To view the specific rates and disallowed amounts calculated for each vehicle in the **Fiscal Report**, enable the :guilabel:`Vehicle Split` filter (located under :icon:`fa-sliders` :guilabel:`Posted Entries`).
```

.. image:: belgium/vehicle-split.png

>    :alt: Disallowed expenses for cars in fiscal report

Reviewer: ```suggestion
   :alt: Disallowed expenses for vehicles in fiscal report
```
